tsdb/index/index.go

```diff
@@ -720,6 +720,7 @@ func (w *Writer) writePostingsOffsetTable() error {
 	}
 
 	// Cleanup temporary file.
+	f.Close() // Needed for Windows.
```

Comment:
Just curious.. why? (:

Reply:
:+1: 

f is mmap of fP0 so it's important to close this handler. I would say it's sane for non windows as well (: